package.json

```diff
@@ -44,7 +46,7 @@
   "devDependencies": {
     "@types/cannon": "^0.1.5",
     "browserify": "^14.4.0",
-    "ts-node": "^8.10.2",
+    "ts-node": "^9.1.1",
```

Comment:
doesnt this imply breaking changes to existing code ? I am not sure if we should update this while changing large parts at the same time.